simulator/src/plotArea.js

```diff
@@ -425,10 +425,10 @@ export function setupTimingListeners() {
     $('.timing-diagram-calibrate').on('click', () => {
         plotArea.calibrate();
     })
-    $('.timing-diagram-resume').on('click', () => {
+    $('.resume-autoscroll').on('click', () => {
         plotArea.resume();
     })
-    $('.timing-diagram-pause').on('click', () => {
+    $('.pause-autoscroll').on('click', () => {
```

Comment:
I think it's better to not change the classnames if it's not required (this might break something we are not aware of). Rest all looks good to me!

Reply:
@gr455 Thanks for reviewing and I have changed the 'class' back to the previous ones...